src/components/forms/sendEmailForm/index.tsx

```diff
@@ -33,9 +34,14 @@ const EmailPreview = styled.div`
 
 interface SendEmailFormProps {
   readonly emails: string[];
+  readonly sendEmailForm: FormInstance<SendEmailRequest>;
 }
 
-const SendEmailForm: React.FC<SendEmailFormProps> = ({ emails }) => {
+const SendEmailForm: React.FC<SendEmailFormProps> = ({
+  emails,
+  sendEmailForm,
+}) => {
+  const SendEmailForm: React.FC<SendEmailFormProps> = ({ emails }) => {
```

Comment:
⛏️ Looks like you have some errors (from merging) - make sure to get rid of this extra declaration of `SendEmailForm`!

Reply:
Also make sure to remove line 50:
```const [sendEmailForm] = Form.useForm();```
since we're passing the form instance as a prop now